layers/+lang/java/packages.el

```diff
@@ -437,6 +441,44 @@
 
         "x:" 'meghanada-run-task))))
 
+(defun java/init-lsp-java ()
+  (use-package lsp-java
+    :defer t
+    :config
+    (progn
+      ;; key bindings
+      (dolist (prefix '(("mc" . "compile")
+                        ("mg" . "goto")
+                        ("mr" . "refactor")))
+      (spacemacs/set-leader-keys-for-major-mode 'java-mode
```

Comment:
`(spacemacs/declare-prefix-for-mode mode "mq" "lsp")` will fix ugly q-prefix with some meaningful label.

Reply:
This "qr" will be removed once #10486 is in.

Reply:
and it was fixed there.